src/components/OnLeavePrompt.js

```diff
@@ -55,28 +42,18 @@ class OnLeavePrompt extends React.Component {
     var realLeave   = leaveText || 'Leave',
         realStay    = stayText || 'Stay',
         realMessage = message;
-    if ( message === 'default' ) { realMessage =  'Selecting "' + realLeave + '" will erase the information you have put into the form. You will still be able to click it after downloading.'; }
```

Comment:
I think it was @knod that added in that last sentence. Might want to check with her before taking it out.

Reply:
@knod does the thumbs up mean keep it out or put it back in?

Reply:
@ethanbb: I was thinking for putting it back in and add your proposed changes. The wording will also need work at some point, but that'll be a task for another time.